nixos/modules/services/x11/unclutter.nix

```diff
@@ -1,32 +1,78 @@
 { config, lib, pkgs, ... }:
+
 with lib;
+
 let cfg = config.services.unclutter;
+
 in {
-  options = {
-    services.unclutter.enable = mkOption {
+  options.services.unclutter = {
+
+    enable = mkOption {
+      description = "Enable unclutter to hide your mouse cursor when inactive";
       type = types.bool;
       default = false;
       example = true;
-      description = "Enable unclutter to hide your mouse cursor when inactive";
     };
 
-    services.unclutter.arguments = mkOption {
-      description = "Arguments to pass to unclutter command";
-      default = "-idle 1";
+    package = mkOption {
```

Comment:
I just thought one may want to use a fork or a custom version. There is also a rewrite of the original package called [unclutter-xfixes](https://github.com/Airblader/unclutter-xfixes). I'm not sure the arguments are compatible though.